Octokit/Models/Request/RepositoryTransfer.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryTransfer
+    {
+        public RepositoryTransfer(string newOwner)
+        {
+            Ensure.ArgumentNotNullOrEmptyString(newOwner, nameof(newOwner));
+
+            NewOwner = newOwner;
+        }
+
+        public RepositoryTransfer(string newOwner, int[] teamId)
+            : this(newOwner)
+        {
+            Ensure.ArgumentNotNullOrEmptyArray(teamId, nameof(teamId));
+
+            TeamId = new int[teamId.Length];
+            Array.Copy(teamId, TeamId, teamId.Length);
+        }
+
+        public string NewOwner { get; set; }
+
+        public int[] TeamId { get; set; }
```

Comment:
We would normally use `IReadOnlyList<int>` rather than a normal `int[]` array

take a look at other request models for examples (eg [here](https://github.com/octokit/octokit.net/blob/master/Octokit/Models/Request/PullRequestReviewRequest.cs))